src/main/resources/assets/hollow/lang/pt_br.json

```diff
@@ -0,0 +1,32 @@
+{
+  "itemGroup.hollow.item_group": "Hollow",
```

Comment:
Item group not renamed. Probably do that.

Reply:
This is the name of the mod. I don't think it should be translated, especially because "oco" as an item group name sounds *odd*.